scripts/testing_scripts/test_all_sandia

```diff
@@ -495,6 +499,21 @@ elif [ "$MACHINE" = "mayer" ]; then
     ARCH_FLAG="--arch=ARMV8_THUNDERX2"
   fi
 
+elif [ "$MACHINE" = "caraway" ]; then
+  SKIP_HWLOC=True
+
+  BASE_MODULE_LIST="cmake/3.19.3,<COMPILER_NAME>/<COMPILER_VERSION>"
+
+  HIPCLANG_BUILD_LIST="Hip_Serial"
+  HIPCLANG_WARNING_FLAGS=""
```

Comment:
why are they no warning flags?

Reply:
I added warning flags to match those from the Jenkins CI jobs, SHA ac76a3054105bb05a59b53a35cbcb38f5037cf7e